tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Steps/OdsApiSteps.cs

```diff
@@ -30,6 +30,11 @@ public async Task CallToTheOdsApiWillFail()
             await AddMapping(CreateMappingModel("/ORD/2-0-0/organisations/*", 500));
         }
 
+        internal async Task SetUpGETChildrenEndpoint(string odsCode, string responseBody)
+        {
+            await AddMapping(CreateMappingModel($"/ORD/2-0-0/organisations", 200, responseBody));
```

Comment:
Is it not worth having the string "/ORD/2-0-0/organisations" as a constant as its used other places (or put in appsettings)